AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2043,6 +2069,19 @@ public static Object objectForKeyPathComponents(Object[] pathComponents) {
     return messages;
   }
 
+  /**
+   * Set this to true if you want details about the variable assignments
+   * on the server.
+   * Default is NO.
+   */
+  public static Map<String, Object> variantDebugInfo() {
```

Comment:
Create and use custom type for `VariantDebugInfo`

Reply:
^